apps/angular/projection/src/app/ui/list-item/list-item.component.ts

```diff
@@ -1,35 +1,17 @@
 import { Component, Input } from '@angular/core';
-import { StudentStore } from '../../data-access/student.store';
-import { TeacherStore } from '../../data-access/teacher.store';
-import { CardType } from '../../model/card.model';
 
 @Component({
   selector: 'app-list-item',
   template: `
     <div class="border border-grey-300 py-1 px-2 flex justify-between">
-      {{ name }}
-      <button (click)="delete(id)">
-        <img class="h-5" src="assets/svg/trash.svg" />
-      </button>
+      <ng-content />
+      <ng-content select="delete-button" />
     </div>
   `,
   standalone: true,
 })
 export class ListItemComponent {
-  @Input() id!: number;
   @Input() name!: string;
-  @Input() type!: CardType;
 
-  constructor(
-    private teacherStore: TeacherStore,
-    private studentStore: StudentStore
-  ) {}
-
-  delete(id: number) {
-    if (this.type === CardType.TEACHER) {
-      this.teacherStore.deleteOne(id);
-    } else if (this.type === CardType.STUDENT) {
-      this.studentStore.deleteOne(id);
-    }
-  }
+  constructor() {}
```

Comment:
don't ass empty constructor, it adds extra bytes into your final bundle for no reason

Reply:
yes i will

Reply:
yes i have removed unused constructors